packages/core/src/icon-shapes/icon.element.scss

```diff
@@ -22,11 +22,12 @@
 }
 
 :host {
+  --color: #{$cds-token-color-neutral-700};
```

Comment:
this was changed because icon colors were not being overwritten in complex components. please verify before merging. check the custom alert icons. it should be a light color.